permiso/src/main/java/com/greysonparrelli/permiso/PermisoDialogFragment.java

```diff
@@ -55,9 +78,15 @@ public void onCreate(Bundle savedInstanceState) {
         // Retain instance state so we can keep our listeners registered after a rotation
         setRetainInstance(true);
 
-        mTitle = getArguments().getString(KEY_TITLE);
-        mMessage = getArguments().getString(KEY_MESSAGE);
-        mButtonText = getArguments().getString(KEY_BUTTON_TEXT);
+        if (getArguments().containsKey(KEY_TITLE)) {
+            mTitle = getArguments().getString(KEY_TITLE);
```

Comment:
```Bundle.getString()``` will return null by default if there's no string matching the key. We don't have to do the contains check. [Docs](https://developer.android.com/reference/android/os/BaseBundle.html#getString(java.lang.String))